docs/source/extension/extension_migration.rst

```diff
@@ -79,6 +79,11 @@ CSS class name change in the ``WindowedList`` superclass of ``StaticNotebook``
 - The notebook scroll container is now ``.jp-WindowedPanel-outer`` rather than ``.jp-Notebook``
 - Galata notebook helpers `getCell` and `getCellCount` were updated accordingly
 
+Visibility of StatusBar elements at high magnifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Statusbar items are by default now hidden at high magnification/low resolution to prevent overlap for those using the application at high magnifications.
```

Comment:
```suggestion
- Statusbar items are now hidden by default at high magnification/low resolution to prevent overlap for those using the application at high magnifications.
```